### PR TITLE
Added scheduler tests, Updated Driver Mock

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by sttts** [comment]
Up to that question above: LGTM.

File: pkg/scheduler/scheduler_test.go [hunk 1]
```diff
@@ -72,3 +79,253 @@ func TestSlaveStorage_getSlaveIds(t *testing.T) {
...
+	//slave hostnames should still be all present
+	assert.Equal(2, len(testScheduler.slaves.getSlaveIds()))
+}
+
+//test we can handle different status updates, TODO check state transitions
+func TestStatus_Update(t *testing.T) {
+
+	mockdriver := MockSchedulerDriver{}
+	// setup expectations
+	mockdriver.On("KillTask", util.NewTaskID("test-task-001")).Return(mesos.Status_DRIVER_RUNNING, nil)
```
Comment (jdef) [inline pkg/scheduler/scheduler_test.go]: probably want to execute `mockdriver.AssertExpectations(t)` to confirm that this was invoked

Comment (joerg84) [inline pkg/scheduler/scheduler_test.go]: thx, added

**Comment by jdef** [comment]
LGTM, @joerg84 is this ready for merge?

**Pull request merged**